Fastfile

```diff
@@ -3,11 +3,16 @@ fastlane_version "1.55.0"
 default_platform :ios
 
   before_all do
+    if ENV['SLACK_URL'] == nil
+      ENV['SLACK_URL'] = ENV['TAB_SLACK_WEBHOOK_URL']
+    end
   end
 
+
   lane :test do
     setup()
-    scan
+    skip_slack = ENV['SCAN_SLACK_CHANNEL'] == "" || ENV['SCAN_SLACK_CHANNEL'] == nil
```

Comment:
Honestly I have no idea on this my ruby isn't very strong, ill take a look tonight and report back